tdrs-backend/tdpservice/users/serializers.py

```diff
@@ -1,8 +1,14 @@
 """Serialize user data."""
```

Comment:
We made updates to the user serializer to give a consistent response with the `authorization-check` and `set-profile` endpoints.